src/components/Landing/Landing.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { signOut } from '../../service/authentication'
+
+const Landing = () => {
+
+    const handleLogout = () => {
+        signOut()
+    }
+    
+    return (
+        <div>
+            <button onClick={handleLogout}>logout</button>
```

Comment:
It would be good to test this user interaction.

Reply:
this solution is temporary, when we have the final destination for the button we will test the functionality